exchange/bidder.go

```diff
@@ -726,3 +729,13 @@ func getBidTypeForAdjustments(bidType openrtb_ext.BidType, impID string, imp []o
 	}
 	return string(bidType)
 }
+
+func getBidderTmax(ctx context.Context, requestTmaxMS int64, tmaxAdjustments *config.TmaxAdjustments) int64 {
+	if tmaxAdjustments != nil && tmaxAdjustments.Enabled {
+		if deadline, ok := ctx.Deadline(); ok {
+			remainingDurationInMS := int64(time.Until(deadline)) / int64(time.Millisecond)
```

Comment:
Rather than worrying about getting the math right, i recommend to use `time.Unitl(deadlne).Milliseconds()`.

Reply:
https://github.com/prebid/prebid-server/pull/2803/commits/33bac30a0402c327df63c0852216618980fc54da makes change to use `Milliseconds()`